gcloud/bigtable/cluster.py

```diff
@@ -317,11 +206,12 @@ def create(self):
         """
         request_pb = _prepare_create_request(self)
         # We expect a `google.longrunning.operations_pb2.Operation`.
-        operation_pb = self._instance._client._instance_stub.CreateCluster(
-            request_pb)
+        client = self._instance._client
+        operation_pb = client._instance_stub.CreateCluster(request_pb)
 
-        op_id = _process_operation(operation_pb)
-        return Operation('create', op_id, cluster=self)
+        operation = Operation.from_pb(operation_pb, client)
+        operation.target = self
+        return operation
```

Comment:
> Can you give an example of what the object looks like after a request? (Or if there are only 4, do all 4?)

The `Operation` object?  The three (not four, that was my error) cases we have in `gcloud.bigtable`:

- `Instance.create` returns an operation created via `Operation.from_pb`:  its `name` with be copied from the `operation_pb.name', its metadata will be deserialized from `operation_pb.metadata`, and thus an instance of `CreateInstanceMetadata`.
- `Cluster.create` returns an operation created via `Operation.from_pb`:  its `name` with be copied from the `operation_pb.name', its metadata will be None (because the `CreateCluster` API does not return any metadata).
- `Cluster.update` returns an operation created via `Operation.from_pb`:  its `name` with be copied from the `operation_pb.name', its metadata will be deserialized from `operation_pb.metadata`, and thus an instance of `UpdateClusterMetadata`.